source/main.cpp

```diff
@@ -354,6 +345,15 @@ int main(int argc, char *argv[])
          * Establish MQTT connection using claim certificates and private key to provision the device/thing.
          */
 #    if !defined(DISABLE_MQTT)
+        /**
+         * init() is currently responsible for making sure only 1 instance of Device Client is running at a given time.
```

Comment:
I think the wording over here is confusing. It should be something like this.. 
```
init() is currently responsible for making sure only 1 MQTT connection is established on Device Client...
```
any thoughts @shangabl